CompilerSource/compiler/components/parse_and_link.cpp

```diff
@@ -224,7 +224,11 @@ int lang_CPP::compile_parseAndLink(const GameData &game, CompileState &state) {
 
   //Done with scripts and timelines.
 
-
+  // Index object ids by name for lookup for collision event name->integer
+  //TODO: could be merged with parsed_objects lookup below
```

Comment:
This TODO should be more about performing these lookups ahead of time (per my old comment), but I'm going to be the one who ends up cleaning this up, and I'll know that much, when I go to handle it.